fetch/fetch.go

```diff
@@ -0,0 +1,174 @@
+package fetch
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+)
+
+var (
+	client *ssm.SSM
+	paths  []string
+	tags   []string
+
+	trueBool = true
+)
+
+func init() {
+	// initialize AWS client
+	initClient()
+}
+
+func initClient() {
```

Comment:
why do this in a separate func?
```suggestion
	// initialize AWS client
```

Reply:
no-look copy pasta